apps/db/priv/repo/migrations/20170812001236_create_player.exs

```diff
@@ -6,6 +6,7 @@ defmodule DB.Repo.Migrations.CreatePlayer do
       add :user_id, references(:users, on_delete: :delete_all)
       add :game_id, references(:games, on_delete: :delete_all)
       add :status, :string, null: false
+      add :is_human, :boolean, null: false
```

Comment:
Might we want to default this one if everyone starts out as human? Also, I realize we put this field in the OG schema, but it's worth noting that this is the first field we're adding to it that is not agnostic with regards to the game itself. Are we ok with this? I don't want to hold us back at all, but given our talks about keeping things extensible it might be worth a discussion. A possible alternative would be for it to live in player stats, but that has implications as well.

Reply:
yea, I totally agree. I'm down to talk further about this. It definitely felt weird adding this here, but I wasnt sure how to extend the player model